docs/tf2_saved_model.md

```diff
@@ -266,3 +266,56 @@ Weight regularizers on individual layers are saved (with their regularization
 strength coefficients), but weight regularization from within the optimizer
 (like `tf.keras.optimizers.Ftrl.l1_regularization_strength=...)`)
 is lost. Advise consumers of your SavedModel accordingly.
+
+### Re-exporting the fine-tuning result
```

Comment:
Hi @meethariprasad, this is high-quality example code, but it is also much, much longer than anything in this documentation. Based on your work, and your verification of this approach, I have created a condensed version https://github.com/tensorflow/hub/pull/445, which IMHO blends in better with the overall flow and verbosity.

Would your suggestion for improvement be met by this condensed version as well?

Reply:
Yes @arnoegw. You have re written it in great way.

Reply:
OK, great! I'll proceed with https://github.com/tensorflow/hub/pull/445 and close this.

Thanks for bringing up this gap in our docs, and validating a solution!